.github/workflows/audit.yml

```diff
@@ -1,6 +1,7 @@
 name: Audit
 
 on:
+  workflow_dispatch:
   schedule:
     - cron: '0 0 * * *'
   push:
```

Comment:
Also can't edit the lines below this one 😂  but I think this should be added to `paths`:

```yml
- "**/package.json"
- "**/yarn.lock"
```